client/components/forms/docs/example.jsx

```diff
@@ -35,7 +36,10 @@ import PhoneInput from 'components/phone-input';
 /**
  * Internal dependencies
  */
-var countriesList = require( 'lib/countries-list' ).forSms();
+const countriesList = require( 'lib/countries-list' ).forSms();
+const currencies = require( 'lib/format-currency/currencies' ).CURRENCIES
```

Comment:
Let's switch to an import in a follow up if we don't need the require here.

So the main difference between `import` and `require` is that imports can be analyzed at compile time, while `require` may be dynamic. If we say wanted to optionally load a module, we'd be forced to use `require`. Another use case is when we need to guarantee execution order, for example in our tests where we use mockery. 

Since we don't have native imports yet, the following is mostly a babel implementation detail, but `require` is synchronous while `import` is async. I wouldn't really rely on this since it may change in the future, but I have run into strange behavior before, where I was forced to switch over to `require`.

see also https://stackoverflow.com/posts/31355283/revisions